test/airbrake/utils_test.exs

```diff
@@ -48,4 +46,36 @@ defmodule Airbrake.UtilsTest do
              }
     end
   end
+
+  describe "detuple/1" do
+    test "returns a map when input is a struct" do
+      input = %Struct{baz: 100, qux: 200}
+
+      assert Utils.detuple(input) == %{baz: 100, qux: 200}
+    end
+
+    test "returns the original data when input is a map" do
+      input = %{baz: 100, qux: 200}
+
+      assert Utils.detuple(input) == input
+    end
+
+    test "returns the original data when input is a list" do
+      input = ["foo", "bar"]
+
+      assert Utils.detuple(input) == input
+    end
+
+    test "returns a list when input is a tuple" do
+      input = {:ok, "something"}
+
+      assert Utils.detuple(input) == [:ok, "something"]
+    end
+
+    test "returns the original value when the input is anything else" do
+      input = "something"
+
+      assert Utils.detuple(input) == input
+    end
```

Comment:
We should have a few tests to make sure we're recursing properly.

This could be multiple tests, but basically we need lists in maps in tuples in struct and a couple permutations of this nesting.